styles.css

```diff
@@ -81,10 +81,13 @@
     width: 100%;
 }
 .navbar, .navbarurl {
```

Comment:
Please fix the linter error by inserting a new line after a comma. You may use [this link](https://github.com/microverseinc/linters-config) to fix all linter errors.

Reply:
- [ ] Good job so far for using the inline comments :+1:  But please phrase the comments politely, as suggestions instead of orders for example instead of "You may use ..." you could have said, "I would recommend you use  ..."  :smile: 

- [ ] Another thing, I would like to recommend you to always tell the reason, why you suggest something or request a change. An example will be:  Please use a semantic html tag .... because the <main> tag specifies the main content of a document. Please fix it :smile: